modules/installation-registry-storage-block-recreate-rollout.adoc

```diff
@@ -32,5 +32,45 @@ $ oc patch config.imageregistry.operator.openshift.io/cluster --type=merge -p '{
 ----
 +
 . Provision the PV for the block storage device, and create a PVC for that volume. The requested block volume uses the ReadWriteOnce (RWO) access mode.
+.. You can define a VMware vSphere PersistentVolumeClaim by creating a file, `pvc.yaml`, with the following contents:
```

Comment:
Small wording suggestion for flow, unless it is important to isolate the specific name of the file:

s/You can define a VMware vSphere PersistentVolumeClaim by creating a file, `pvc.yaml`, with the following contents:/You can define a VMware vSphere PersistentVolumeClaim by creating a `pvc.yaml` file with the following contents:/